packages/client/src/utils/styleable.js

```diff
@@ -6,7 +6,9 @@ import { builderStore } from "stores"
 const buildStyleString = (styleObject, customStyles) => {
   let str = ""
   Object.entries(styleObject || {}).forEach(([style, value]) => {
-    if (style && value != null) {
+    if (style === "background-image-url" && value != null) {
+      str += `background-image: url("${value}"); background-repeat: no-repeat; background-size: cover; `
```

Comment:
Hm, this seems like quite a dangerous place to do this - is it not possible to add this directly into the container component to encapsulate a bit?

Reply:
I had a go at moving this code into Container.svelte but I wasn't able to get it working. I'm not sure of the order in which the styling is set.

Maybe @aptkingston has an opinion?

Reply:
Sorry I was completely sure I had replied to this...

I would prefer not to add in this sort of edge case, but it's reasonably safe if we need to. I would probably rather extend the style definitions to support something like a `format` key, so for URL you could define it as:
```js
    {
      control: Input,
      label: "Image URL",
      key: "background-image",
      format: url => url ? `url("${url}"); background-repeat: no-repeat; background-size: cover;` : ""
    },
```
Which would then require no changes to `styleable.js` and also be reusable in future.